### PR TITLE
snap, wrappers: add per-app environment file

### PR DESCRIPTION
**Pull request by mvo5**
This branch adds an environment key on each application (in addition to the global one) as discussed during the sprint.

### CODE REVIEW

**Comment by mvo5** [comment]
retest this please

File: snap/info_snap_yaml.go [hunk 1]
```diff
@@ -178,6 +180,7 @@ func InfoFromSnapYaml(yamlData []byte) (*Info, error) {
 			SocketMode:      yApp.SocketMode,
 			ListenStream:    yApp.ListenStream,
 			BusName:         yApp.BusName,
+			Environment:     yApp.Environment,
```
Comment (pedronis) [inline snap/info_snap_yaml.go]: it's a bit strange (and makes the reader wonder) doing a copy for the global one, but reusing the one from yaml parsing here, I suppose you could reuse the global one too?

**Comment by pedronis** [comment]
looks good, one wondering

File: snap/info_snap_yaml.go (hunk 1 above)
Comment (mvo5) [inline snap/info_snap_yaml.go]: Thanks, good catch. There is indeed no need to copy this.

**Comment by mvo5** [comment]
Closing for now, Gustavo suggested some changes for this.

**Pull request closed without merging**